deploy_pgcluster.yml

```
@@ -190,6 +190,9 @@
     - role: postgresql-extensions
       when: is_master == "true" and postgresql_extensions | length > 0
 
+    - role: postgresql-initial-db
```

```suggestion
    - role: postgresql-dump-restore
```

I understand the purpose of this role is to recover data from a dump